rubygem-smart_proxy_openscap/rubygem-smart_proxy_openscap.spec

```diff
@@ -3,10 +3,11 @@
 %global foreman_proxy_bundlerd_dir %{_datadir}/foreman-proxy/bundler.d
 %global foreman_proxy_pluginconf_dir %{_sysconfdir}/foreman-proxy/settings.d
 %global spool_dir %{_var}/spool/foreman-proxy/openscap
+%global content_dir %{_var}/lib/openscap
```

Comment:
It is related to foreman-proxy and I wouldn't mind creating 'openscap/content' directory under foreman-proxy's directory.
Please also note that openscap is using /var/lib/openscap and creates different directories (like xccdf_scans, etc) - so I thought of adding a new directory (content) and save scap xml files over there - but that it not a must.
Waiting your guidance on what is preferred:
/var/lib/openscap/content symlinked to /path/to/proxy/openscap/content
-or-
just create /path/to/proxy/openscap/content
Thanks.